styles.css

```diff
@@ -1,7 +1,32 @@
-body {
-  background: #4ea0e8;
+.clear {
+  float:clear
+}
+img {
+  max-height:50px;
+  display:inline;
+}
+nav li {
+  display:inline;
+  padding: 5%;
+  width: 32%
+}
+nav {
+    width: 100%;
+    text-align:center;
+
 }
+header {
+  color:black;
+  border: 3px solid red;
+  float:clear;
```

Comment:
Not sure what the goal is here. [`float` property](https://developer.mozilla.org/en-US/docs/Web/CSS/float) would be `left`, `right`, or `none`; [`clear` property](https://developer.mozilla.org/en-US/docs/Web/CSS/clear) can be `left`, `right`, `both`, or `none`.